pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -44,6 +49,24 @@ const (
 	taskRunControllerName = "TaskRun"
 )
 
+const (
+	descScheduling       = "scheduling"
+	descInitializing     = "initializing"
+	descRunning          = "running"
+	descSucceeded        = "succeeded"
+	descFailed           = "failed"
+	descUnknown          = "unknown status"
+	descMissingCondition = "missing end condition"
+)
+
+const (
```

Comment:
This should be `var` instead of `const`. Its making the code not compile and thats why the tests in the PR are failing

Reply:
i've changed this back to var